04-deny-traffic-from-other-namespaces.md

```diff
@@ -50,36 +50,35 @@ networkpolicy "deny-from-other-namespaces" created"
 
 Note a few things about this manifest:
 
-- `namespace: secondary` deploys it to the `secondary` namespace.
-- it applies the policy to ALL pods in `secondary` namespace as the
+- `namespace: default` deploys it to the `default` namespace.
+- it applies the policy to ALL pods in `default` namespace as the
   `spec.podSelector.matchLabels` is empty and therefore selects all pods.
-- it allows traffic from ALL pods in the `secondary` namespace, as
+- it allows traffic from ALL pods in the `default` namespace, as
    `spec.ingress.from.podSelector` is empty and therefore selects all pods.
 
 ## Try it out
 
-Query this web service from the `default` namespace:
+Query this web service from the `foo` namespace:
 
 ```sh
-$ kubectl run test-$RANDOM --namespace=default --rm -i -t --image=alpine -- sh
-/ # wget -qO- --timeout=2 http://web.secondary
+$ kubectl run test-$RANDOM --namespace=foo --rm -i -t --image=alpine -- sh
```

Comment:
I believe you need to create `foo` namespace first.
Also need to clean it up at the end.

Reply:
@cscetbon please add the following line

kubectl create namespace default

before line 64

Reply:
@boredabdel I think you mean `foo` as `default` always exists

Reply:
Yes sorry foo